schema/me/update_conversation.js

```diff
@@ -15,20 +16,27 @@ export default mutationWithClientMutationId({
     buyer_outcome: {
       type: new GraphQLNonNull(BuyerOutcomeTypes),
     },
-    id: {
-      type: new GraphQLNonNull(GraphQLString),
+    ids: {
+      type: new GraphQLList(GraphQLString),
     },
   },
-  outputFields: ConversationFields,
-  mutateAndGetPayload: ({ buyer_outcome, id }, request, { rootValue: { accessToken } }) => {
+  outputFields: {
+    conversations: {
```

Comment:
Here we return a single `conversations` Array of all the modified conversations. We just resolve the whole body (since it's coming from the `mutateAndGetPayload` below (which is returning a list of all conversations).